src/core/Table/Table.tsx

```diff
@@ -129,6 +129,11 @@ export type TableProps<
    * If not specified, all rows are enabled.
    */
   isRowDisabled?: (rowData: T) => boolean;
+  /**
+   * Function that should return custom props passed to the each row.
+   * Must be memoized.
+   */
+  rowProps?: (rowData: T) => React.ComponentPropsWithRef<'div'>;
```

Comment:
hm... you sure about this type?
I see for simple elements examples with `React.DetailedHTMLProps`, just it is really long to write..

Reply:
Previously I had `React.HTMLAttributes<HTMLDivElement> & React.RefAttributes<HTMLDivElement>`. Mayank recommended current one: https://react-typescript-cheatsheet.netlify.app/docs/advanced/patterns_by_usecase/#wrappingmirroring-a-html-element.
What `React.DetailedHTMLProps` gives? Because to achieve the same, I have to write: `React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>`

Reply:
Yeah, not sure.. I could not find quickly good info about types.. Just I see `ComponentProps` usually used for components, rather than native elements. But I guess its fine.